gheb/docusign-bundle/5.0/config/packages/docusign.yaml

```diff
@@ -0,0 +1,38 @@
+docusign:
+    demo: "%kernel.debug%"
+    mode: embedded # Mode used to sign (remote or embedded)
+
+    # Authentication credentials to log into docusign.
+    auth_jwt:
+        private_key: "%kernel.project_dir%/%CONFIG_DIR%/jwt/docusign.pem"
+        integration_key: "51b176de-3298-460b-8201-f73e323eee14" # Replace by yours
+        user_guid: "51b176de-3298-460b-8201-f73e323eee14" # Replace by yours
```

Comment:
These should use environment variables (as comments)

Reply:
Well to be honest at first they were set as such but using validation within the configuration prevented their usage. But in the end, it make more sense for this kind of data to be environment variables (secrets) so I backed up a little on the code by removing the checks.
So yes, I can change that back.

Thanks for the review @Nyholm !